schema/google/showcase/v1beta1/sequence.proto

```diff
@@ -61,14 +61,14 @@ service SequenceService {
     option (google.api.method_signature) = "name";
   };
 
-    // Attempts a streaming sequence.
-    rpc AttemptStreamingSequence(AttemptStreamingSequenceRequest) returns (google.protobuf.Empty) {
-      option (google.api.http) = {
-        post: "/v1beta1/{name=sequences/*}"
-        body: "*"
-      };
-      option (google.api.method_signature) = "name";
+  // Attempts a streaming sequence.
+  rpc AttemptStreamingSequence(AttemptStreamingSequenceRequest) returns (google.protobuf.Empty) {
```

Comment:
If this is a server streaming RPC, it needs to have the `stream` keyword in the response type ([docs](https://grpc.io/docs/what-is-grpc/core-concepts)).
```suggestion
  rpc AttemptStreamingSequence(AttemptStreamingSequenceRequest) returns (stream google.protobuf.Empty) {
```